source/Calamari.Tests/KubernetesFixtures/Terraform/Clusters/EKS/kubeconfig.tpl

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1

```

Comment:
@gb-8 has suggested sharing this file between each provider as all three files are the same.

Reply:
@gb-8 I managed to do this but I had to update the code to copy it from the new location.

Reply:
Turns out we only need this for EKS, so the other ones have now been removed.